buildbot/check-sycl.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+PROJECT=
+BRANCH=
+BUILD_NUMBER=
+PR_NUMBER=
+
+# $1 exit code
+# $2 error message
+exit_if_err()
+{
+    if [ $1 -ne 0  ]; then
+        echo "Error: $2"
+        exit $1
+    fi
+}
+
+unset OPTIND
+while getopts ":p:b:r:n:" option; do
+    case $option in
+        p) PROJECT=$OPTARG ;;
+        b) BRANCH=$OPTARG ;;
+        n) BUILD_NUMBER=$OPTARG ;;
+        r) PR_NUMBER=$OPTARG ;;
+    esac
+done && shift $(($OPTIND - 1))
+
+# we're in llvm.obj dir
+BUILD_DIR=${PWD}
+
+make check-sycl VERBOSE=1 LIT_ARGS="-v -j `nproc`"
```

Comment:
There is no difference between `check-sycl`, `check-llvm`, `check-llvm-spirv` and `check-clang` scripts between except the target name. I think it should be one script with configurable target.

Reply:
@vladimirlaz could you please help comment on this. I think one of the advantage of seperate script is easy customization. Anyway, we can combine them if needed.

Reply:
Good catch from my perspective. We can have check.sh script which accepts exact target as command line and environment variable